sandbox/tests/test scenes/alpha/01 - opaque red over opaque blue.appleseed

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- File generated by appleseed.dll version 1.7.0-beta (SSE 4.2). -->
+<!-- File generated by appleseed.dll version 1.8.0-beta (SSE 2). -->
```

Comment:
It just so happens that my current build (which I used to automatically update all the test scenes) doesn't have SSE 4.2 enabled. Not a big deal.